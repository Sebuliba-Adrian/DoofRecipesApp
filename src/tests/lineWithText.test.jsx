import React from "react";
import renderer from "react-test-renderer";
import LineWithText from "../components/authentication/lineWithText";

const props = {
  lineText: "login"
};

it("renders correctly", () => {
  const lineWithText = renderer.create(<LineWithText {...props} />).toJSON();
  expect(lineWithText).toMatchSnapshot();
});
