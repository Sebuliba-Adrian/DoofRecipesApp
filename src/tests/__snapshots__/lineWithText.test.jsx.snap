// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="line-with-text font-weight-bold mb-4"
>
  <div
    className="line"
  />
  <div
    className="line-text"
  >
    login
  </div>
  <div
    className="line"
  />
</div>
`;
